x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/monitoring/exporter/MonitoringDoc.java

```diff
@@ -123,7 +123,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
      * @return a string representing the timestamp
      */
     public static String toUTC(final long timestamp) {
-        return new DateTime(timestamp, DateTimeZone.UTC).toString();
```

Comment:
joda toString:
     ` Output the date time in ISO8601 format (yyyy-MM-ddTHH:mm:ss.SSSZZ).`
when java-time
     ` {@code 2007-12-03T10:15:30+01:00[Europe/Paris]}.`
but it also says:      ` The output is compatible with ISO-8601 if the offset and ID are the same.`
The quick test for sample long shows that the format is the same
`Instant.ofEpochMilli(123).atZone(ZoneOffset.UTC).toString()`
`new DateTime(123, DateTimeZone.UTC).toString();`
`1970-01-01T00:00:00.123Z`

Reply:
can we make this more explicit and not rely on the `toString()` method of classes we dont have control over (even though I assume this does not get changed) Using a `strict_date_time` formatter might just work as expected?